modules/porous_flow/include/utils/PorousFlowCapillaryPressureVG.h

```diff
@@ -0,0 +1,72 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef POROUSFLOWCAPILLARYPRESSUREVG_H
+#define POROUSFLOWCAPILLARYPRESSUREVG_H
+
+#include "MooseTypes.h"
+
+/**
+ * van Genuchten capillary pressure for multiphase flow in porous media.
+ *
+ * Based on van Genuchten, M. Th., A closed for equation for
+ * predicting the hydraulic conductivity of unsaturated soils,
+ * Soil Sci. Soc., 44, 892-898 (1980).
+ */
+namespace PorousFlowCapillaryPressureVG
+{
+  /**
```

Comment:
Namespaces are _not_ indented (http://mooseframework.org/wiki/CodeStandards/). Sorry, this must drive you guys crazy.